hil-test/tests/sha.rs

```diff
@@ -278,7 +270,6 @@ mod tests {
                     block!(digest.save(&mut sha1));
                     all_done = false;
                 }
-                #[cfg(not(feature = "esp32"))]
```

Comment:
I'll be surprised and slightly peeved (because trm lied to me) if this HIL test passes

Reply:
The whole test case is !esp32

Reply:
Ohhhhh gotcha